### PR TITLE
Add "Open Dashboard" button to Immersive Editor

### LINKED ISSUE #3760: Add the dashboard button to the immersive editor
Add the dashboard in the immersive editor.

image

### PR DESCRIPTION
**Pull request by cstns**
## Description

adding a dashboard button and e2e tests

## Related Issue(s)

closes #3760 

## Checklist

 - [ ] ~~I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)~~
 - [ ]  ~~Suitable unit/system level tests have been added and they pass ~~
 - [ ]  ~~Documentation has been updated~~
    - [ ]  ~~Upgrade instructions~~
    - [ ]  ~~Configuration details~~
    - [ ]  ~~Concepts~~
 - [ ]  ~~Changes `flowforge.yml`?~~
    - [ ]  ~~Issue/PR raised on `FlowFuse/helm` to update ConfigMap Template~~
    - [ ]  ~~Issue/PR raised on `FlowFuse/CloudProject` to update values for Staging/Production~~

## Labels

 - [ ]  ~~Backport needed? -> add the `backport` label~~
 - [ ]  ~~Includes a DB migration? -> add the `area:migration` label~~

### CODE REVIEW

File: frontend/src/pages/instance/Editor/index.vue [hunk 1]
```diff
@@ -80,6 +84,8 @@ import ResizeBar from './components/drawer/ResizeBar.vue'
 export default {
     name: 'InstanceEditor',
     components: {
+        DashboardLink,
+        FfTabs,
```
Comment (joepavitt) [inline frontend/src/pages/instance/Editor/index.vue:88]: No need to include `FfTabs` here as it's a global component (as are all `/ui-components`)

File: test/e2e/frontend/cypress/tests/instances/editor.spec.js [hunk 2]
```diff
@@ -11,12 +11,11 @@ describe('FlowForge - Instance editor', () => {
                 const instance = response.body.projects.find(
                     (project) => project.name === instanceName
                 )
-                instance.name = 'qweqwe'
                 cy.visit(`/instance/${instance.id}/`)
             })
     }
 
-    it('Preserves the initial behavior if uses an unsupported launcher version', () => {
+    it.only('Preserves the initial behavior if uses an unsupported launcher version', () => {
```
Comment (joepavitt) [inline test/e2e/frontend/cypress/tests/instances/editor.spec.js:18]: think you've left this in by mistake

Comment (cstns) [reply test/e2e/frontend/cypress/tests/instances/editor.spec.js:18]: indeed I did

**Review by joepavitt — APPROVED** [review]

**Comment by joepavitt** [comment]
Approved subject to resolving conflicts

File: test/e2e/frontend/cypress/tests/instances/editor.spec.js [hunk 3]
```diff
@@ -85,7 +85,6 @@ describe('FlowForge - Instance editor', () => {
                 const instance = response.body.projects.find(
                     (project) => project.name === 'instance-1-1'
                 )
-                instance.name = 'qweqwe'
```
Comment (ZJvandeWeg) [inline test/e2e/frontend/cypress/tests/instances/editor.spec.js:88]: 😄

Comment (cstns) [reply test/e2e/frontend/cypress/tests/instances/editor.spec.js:88]: qweqwe will be the death of me :sweat_smile:. I caught it in another pf and removed it

Comment (joepavitt) [reply test/e2e/frontend/cypress/tests/instances/editor.spec.js:88]: Who's the `git blame` on that? Name and shame!

Comment (cstns) [reply test/e2e/frontend/cypress/tests/instances/editor.spec.js:88]: That would be yours truly :raised_hand_with_fingers_splayed: :1st_place_medal:

**Pull request merged**